dataland-internal-storage/src/test/kotlin/db/migration/V23__MigratePageZeroToNullTest.kt

```diff
@@ -0,0 +1,47 @@
+package db.migration
+
+import db.migration.utils.TestUtils
+import org.junit.jupiter.api.Test
+
+class V23__MigratePageZeroToNullTest {
+
+    @Test
+    fun `check migration script for page set to null where page is 0`() {
+        TestUtils().testMigrationOfSingleDataset(
+            "sfdr",
```

Comment:
I would prefer one complete example per framework from prod or test that covers both cases (value is an in and value is a string).